doc/apimanager-reference.md

```diff
@@ -326,3 +326,15 @@ existing secret names.
 | --- | --- | --- |
 | AWS_ACCESS_KEY_ID | AWS Access Key ID to use in S3 Storage for System's file storage | N/A |
 | AWS_SECRET_ACCESS_KEY | AWS Access Key Secret to use in S3 Storage for System's file storage | N/A |
+
+#### system-smtp
+
+| **Field** | **Description** | **Default value** |
+| --- | --- | --- |
+| address | Address (hostname or IP) of the remote mail server to use. If set to a value different than `""` System will use the mail server to send mails related to events that happen in the API management solution |  `""` |
+| username | In case the mail server requires authentication, set the username using this setting | `""` |
+| password | In case the mail server requires authentication, set the password using this setting | `""` |
+| domain | If a HELO domain is needed to be specified set it using this setting | `""` |
+| port | Port of the remote mail server to use | `""` |
+| authentication | In case the mail server requires authentication, set this setting to the authentication type here. `plain` to send the password in the clear, `login` to send password Base64 encoded, or `cram_md5` to combine a Challenge/Response mechanism to exchange information and a cryptographic Message Digest 5 algorithm to hash important information | `""` |
```

Comment:
```suggestion
| authentication | Specify the authentication type in case the mail server requires authentication. Use `plain` to send the password in the clear, `login` to send password Base64 encoded, or `cram_md5` to combine a Challenge/Response mechanism to exchange information and a cryptographic Message Digest 5 algorithm to hash important information | `""` |
```

Reply:
But "or `cram_md5` to combine a Challenge/Response mechanism to exchange information and a cryptographic Message Digest 5 algorithm to hash important information" is very hard to read, can we simplify?

Reply:
would you use bother username and password fields when picking `cram_md5`?

Reply:
That description is the one from the official rubydocs documentation. Anyway, I've changed it to '`cram_md5` to combine a Challenge/Response mechanism based on the HMAC-MD5 algorithm '

If you don't need to set username and password then you don't have to in that case.
I've updated the username and password fields to
"In case the mail server requires authentication and the authentication type requires it"
instead of just
"In case the mail server requires authentication"